templates/graphql.yml

```diff
@@ -53,4 +53,4 @@ dataSources:
   - type: AWS_LAMBDA
     name: getDisease
     config:
-      functionName: getDisease
+      lambdaFunctionArn: { Fn::GetAtt: [GetDiseaseLambdaFunction, Arn] }
```

Comment:
@bboure is this the best way to do this?

Reply:
@bboure no, the best way is as it was. Use `functionName: getDisease` where the value is the logical name (key) of the function, [here](https://github.com/RareCamp/RareCamp/blob/master/templates/functions.yml#L1)